3.9-minimal/Dockerfile.rhel8

```diff
@@ -88,8 +99,7 @@ RUN \
     ${APP_ROOT}/bin/pip install /opt/wheels/pip-* && \
     rm -r /opt/wheels && \
     chown -R 1001:0 ${APP_ROOT} && \
-    fix-permissions ${APP_ROOT} -P && \
-    rpm-file-permissions
```

Comment:
`rpm-file-permissions` are not needed anymore for some reason?

Reply:
I thought that it's no longer needed but it seems that it'd be better to have it here. I've added it. Thanks for spotting this!

Reply:
No worries! :+1: